yambol/yambol/plugins/sql.py

```
@@ -0,0 +1,51 @@
+    for t in self.db.types:
+      types_idx[t.name] = t
+    top = []
+    before = []
+    creates = []
+    after = []
+    bottom = []
+    for t in self.db.tables:
+      create = []
+      for f in t.fields:
+        print(f)
+        custype = f['type'] if 'type' in f else f.type.name
```

Ruff (0.8.2)

Use `f.get('type', f.type.name)` instead of an `if` block

Replace with `f.get('type', f.type.name)`

(SIM401)